### PR TITLE
Add several tests and fix wording issues

### PR DESCRIPTION
**Pull request by pgilad**

### CODE REVIEW

**Comment by wraithgar** [comment]
This is all readme/test updates, no actual code changing. Merging, and it will deploy when https://github.com/AmpersandJS/ampersand-state/pull/150 ships.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # ampersand-state
 
 <!-- starthide -->
-Part of the [Ampersand.js toolkit](http://ampersandjs.com) for building clientside applications.
+Part of the [Ampersand.js toolkit](http://ampersandjs.com) for building client side applications.
```
Comment (wraithgar) [inline README.md]: All the other readmes (i.e. https://github.com/AmpersandJS/ampersand-model) say clientside.

**Comment by wraithgar** [comment]
Will wait to merge till client side is reverted then, I guess.

**Comment by pgilad** [comment]
Updated back to `clientside` ;)

**Pull request merged**